var/spack/repos/builtin/packages/legion/package.py

```diff
@@ -414,9 +419,6 @@ def cmake_args(self):
             options.append("-DLegion_BUILD_BINDINGS=ON")
             options.append("-DLegion_REDOP_COMPLEX=ON")  # required for bindings
 
-        if spec.variants["build_type"].value == "Debug":
-            cmake_cxx_flags.extend(["-DDEBUG_REALM", "-DDEBUG_LEGION", "-ggdb"])
-
```

Comment:
I agree that the removed code seems odd. I did not put it here and don't know who did. I mainly want to be sure we're not breaking anything. @pmccormick do you know?

Reply:
AFAICT, the `cmake_cxx_flags` variable exists nowhere in Spack, and it appears to be a local variable, so it cannot have any effect on the build. Nevertheless, it would be good for somebody else to verify it.

Reply:
Hmmm...  I don't recall it off the top of my head either.  (could be that I've forgotten about it ;-) 

In general, I agree -- seems to have zero impact on the build...